WcaOnRails/app/models/competition.rb

```diff
@@ -1122,6 +1122,10 @@ def self.search(query, params: {}, managed_by_user: nil)
     competitions.includes(:delegates, :organizers).order(start_date: :desc)
   end
 
+  def all_activities
+    competition_venues.includes(venue_rooms: { schedule_activities: [:child_activities] }).map(&:all_activities).flatten
```

Comment:
Would this catch deeply-nested activities?  For example

>333-r1
>>333-r1-g1
>>>333-r1-g1-a1

(I don't think there's any reason someone would do that, but technically WCIF supports it I think)

Reply:
Oh, I thought it would, but apparently it only gets the top-level activities, good catch! It's definitely a real scenario, as you most likely assign people to group activities, being nested under round activities. I will fix that!